tianshou/policy/base.py

```diff
@@ -150,6 +152,8 @@ def compute_episodic_return(
         for i in range(len(rew) - 1, -1, -1):
             gae = delta[i] + m[i] * gae
             returns[i] += gae
+        if rew_norm and not np.isclose(returns.std(), 0):
```

Comment:
maybe add an explicit tolerance for `isclose`. By default it is not really restrictive. Something like rtol=1e-04, atol=1e-6 is already very small.